mangaki/mangaki/factories.py

```diff
@@ -21,7 +21,6 @@ class Meta:
 
     username = factory.Faker('user_name')
     email = factory.LazyAttribute(lambda o: '{}@mangaki.fr'.format(o.username))
-    profile = factory.RelatedFactory(ProfileFactory, 'user')
```

Comment:
Actually this is the wrong way to go about it I believe, updating the patch to use `factory.django.mute_signals`.